readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
@kof well, every interpolation there can be a function. So yea, even for properties it could be a function unfortunately :/